game/src/info/trip.rs

```diff
@@ -386,6 +389,329 @@ pub fn cancelled(ctx: &mut EventCtx, app: &App, id: TripID) -> Widget {
     Widget::col(col)
 }
 
+/// Highlights intersections which were "slow" on the map
+fn highlight_slow_intersections(app: &App, details: &mut Details, id: TripID, ctx: &EventCtx) {
+    let data_opt = &app.primary.sim.get_analytics().trip_intersection_delays(id);
+    if data_opt.is_some() {
+        let data = data_opt.unwrap();
+        for (id, time) in data {
+            let intersection = app.primary.map.get_i(id.parent);
+            // Maybe alter the delay times
+            let (normal_delay_time, slow_delay_time) = if intersection.is_traffic_signal() {
+                (30, 120)
+            } else {
+                (5, 30)
+            };
+            let (fg_color, bg_color) = if time < &normal_delay_time {
```

Comment:
Style: It feels weird to compare pointers; in this situation, I think `*time < normal_delay_time` is more clear

Reply:
Yup, that makes sense. Changed.